spec/support/rbac_helper.rb

```diff
@@ -0,0 +1,30 @@
+module Spec
+  module Support
+    module RBACHelper
+      def initial_setup
+        EvmSpecHelper.local_miq_server
+      end
+
+      def login_as(user = @user)
+        User.current_user = user
+      end
+
+      def setup_user_with_tenant
+        initial_setup
+
+        @tenant = FactoryGirl.create(:tenant)
+        @group = FactoryGirl.create(:miq_group, :tenant => @tenant)
+        @user = FactoryGirl.create(:user, :miq_groups => [@group])
+        login_as
+        @user
+      end
+
+      def tag_group_with(tags)
+        @group.entitlement = Entitlement.new
+        @group.entitlement.set_managed_filters([tags])
+        @group.entitlement.set_belongsto_filters([])
+        @group.save!
+      end
+    end
+  end
+end
```

Comment:
Do you REALLY need the instance variables in this module?

Reply:
I'm pretty sure `let` can be used here, maybe with a single `let!`.

Reply:
Forgot the keyword: https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-context